minesweeper.py

```diff
@@ -130,8 +130,16 @@ def timer(self):
                 self.game_time += datetime.timedelta(seconds=1)
                 root.after(1000, self.timer)
             elif self.game_status == GameStatus.game_won:
-                with open("high_scores.json", "w") as high_scores:
-                    json.dump(self.game_time.strftime("%M:%S"), high_scores)
+                with open(where_this_file_is / 'high_scores.json', "w") as score_list:
+                    
+                    json_dict["high_scores"].append({
+                        'time' : self.game_time.strftime("%M:%S"),
```

Comment:
I guess this should be seconds as a number, instead of minutes and seconds as a string? You can't sort `"minutes:seconds"` strings easily, for example. There's this code in `highscores()` function that you can use:

```
converted_to_seconds = mins * 60 + secs
```

Reply:
so, something like this:

```
'time': self.game_time.minute*60 + self.game_time.second,
```

Reply:
Unfortunately datetime.datetime is not serializable in json. Therefore I ended up adding logic for time conversion from m:s to s in highscores function (when json is loaded).